enterprise/views.py

```diff
@@ -499,7 +503,23 @@ def get_enterprise_course_enrollment_page(self, request, enterprise_uuid, course
             logger.error('Unable to find course details for course ID: %s', course_id)
             raise Http404
 
-        platform_name = configuration_helpers.get_value('PLATFORM_NAME', settings.PLATFORM_NAME)
+        enterprise_customer = get_enterprise_customer_or_404(enterprise_uuid)
+
+        enrollment_client = EnrollmentApiClient()
+        enrolled_course = enrollment_client.get_course_enrollment(request.user.username, course_id)
+        enrolled_course_id = enrolled_course['course_details']['course_id']
```

Comment:
I think this will blow up if the user is not enrolled in the given course right? Can we do `enrolled_course.get('course_details', {}).get('course_id')`? And then just `if (enrolled_course_id or ...` below?